src/containers/leaderboard/Leaderboard.js

```diff
@@ -150,21 +151,31 @@ class Leaderboard extends React.Component {
 
   renderRoundsBlocks() {
     const rounds = [];
-    for(let i = NUMBER_OF_ROUNDS; i >= 1; i--) {
+    const currentTotalRound = this.state.round ? this.state.round.total : 0;
+    for(let i = currentTotalRound; i >= 1; i--) {
+      if ( i > (currentTotalRound - MAX_DISPLAYED_TABS)) {
+        rounds.push(
+          <a
+            href={null}
+            key={i}
+            onClick={() => this.selectRound(i)}
+            className={classNames('block__top-switch', 'ratings-traders', {active: this.state.selectedRound === i})}>
+            <FormattedMessage
+              id="leaderboard.round"
+              defaultMessage="ROUND {count}"
+              values={{count: i}}
+            />
+          </a>
+        );
+      }
+    }
+    if (currentTotalRound > MAX_DISPLAYED_TABS) {
       rounds.push(
-        <a
-          href={null}
-          key={i}
-          onClick={() => this.selectRound(i)}
-          className={classNames('block__top-switch', 'ratings-traders', {active: this.state.selectedRound === i})}>
-          <FormattedMessage
-            id="leaderboard.round"
-            defaultMessage="ROUND {count}"
-            values={{count: i}}
-          />
-        </a>
+        <RoundSelect onSelectClick={RoundNumber => this.selectRound(RoundNumber)}
+          rounds={[...Array(currentTotalRound - MAX_DISPLAYED_TABS).keys()].map(x => ++x).reverse()}/>
```

Comment:
Use [lodash.times](https://lodash.com/docs/4.17.10#times) util. Import `lodash.times` package for it.

Example:

```javascript
const offset = 4;
const items = times(5, (i) => offset + 1 + i); // -> [5, 6, 7, 8, 9]
```